module/tools/SystemConfiguration/data/system/default/etc/udev/rules.d/10-nusense.rules

```diff
@@ -0,0 +1 @@
+KERNEL=="ttyACM*", SUBSYSTEM=="tty", SUBSYSTEMS=="usb", ATTRS{manufacturer}=="STMicroelectronics", ATTRS{idProduct}=="5740", ATTRS{idVendor}=="0483", SYMLINK+="nusense", GROUP="uucp", MODE="0666"
```

Comment:
What is the mode before you set it here?

Reply:
it did not have a mode specified, I added this in as a new thing

Reply:
I mean when you just plug the device in without the udev rules file, what is the mode that is automatically assigned to `/dev/ttyACM0`?

Reply:
tried to mimic what `10-nubots.rules` had

Reply:
not sure what the c maps to but 

```
nubots@nugus2 ~ ❯❯❯ ls -lah /dev/ttyACM0                                                                                                                                                             ✘ 130 
crw-rw-rw- 1 root uucp 166, 0 May 16 11:04 /dev/ttyACM0
```
which is something - 666 afaik?

Reply:
Well if you have that udev rule in now it'll always be 666

I'd imagine 660 was what it was before but can't be sure

Reply:
sounds like you dont need to set the mode at all then?

Reply:
Given that previously it was accessible by root and the uucp group and not everyone that would mean the last number must have been 0 but the first two must have been 6s

Reply:
And if it was 660 you also dont need to set the mode

Reply:
works :+1: